src/app/shared/components/wrapper/wrapper.component.scss

```diff
@@ -0,0 +1,15 @@
+:host {
+  height: 100vh;
```

Comment:
I think min-height, should be better here

Reply:
fixed